File: frontend/src/app/fizioterapeut/fizioterapeutUpdateRequest.ts
export interface FizioterapeutUpdateRequest {
    zaposlenikId: number
    imePrezime: string
    jmbg: string
    datumRodjenja: string
    datumZaposlenja: string
    nalogId: number | null
    poslovnaPozicijaId: number
    oblastFizijatrije: string
}

File: DomZaStaraLicaApi/Endpoints/Fizioterapeut/Update/FizioterapeutUpdateRequest.cs
namespace DomZaStaraLicaApi.Endpoints.Fizioterapeut.Update
{
    public class FizioterapeutUpdateRequest
    {
        public int ZaposlenikId { get; set; }
        public string ImePrezime { get; set; }
        public string JMBG { get; set; }
        public DateTime DatumRodjenja { get; set; }
        public DateTime DatumZaposlenja { get; set; }
        public int? NalogId { get; set; }
        public int PoslovnaPozicijaId { get; set; }
        public string OblastFizijatrije { get; set; }
    }
}
